packages/react/.storybook/main.js

```diff
@@ -7,26 +7,36 @@
 
 'use strict';
 
+import remarkGfm from 'remark-gfm';
 const fs = require('fs');
 const glob = require('fast-glob');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
 
+const storyGlobs =
```

Comment:
Could you add a brief comment here explaining why we conditionally exclude some globs when this flag is false?